pages/_app.tsx

```diff
@@ -34,13 +35,14 @@ export default class MyApp extends App {
   }
 
   render() {
+    const isOpenClimbing = PROJECT_ID === 'openclimbing';
     const { Component, pageProps } = this.props as any;
     const { userThemeCookie } = pageProps;
 
     return (
       <>
         <Head>
-          <title>OsmAPP</title>
+          <title>{isOpenClimbing ? 'openclimbing.org' : 'OsmAPP'}</title>
```

Comment:
Stačí použít tuhle konstantu:
```suggestion
          <title>{PROJECT_NAME}</title>
```